core/services/gateway/handlers/functions/subscriptions/subscriptions.go

```diff
@@ -137,8 +137,12 @@ func (s *onchainSubscriptions) queryLoop() {
 		defer cancel()
 
 		latestBlockHeight, err := s.client.LatestBlockHeight(ctx)
-		if err != nil || latestBlockHeight == nil {
-			s.lggr.Errorw("Error calling LatestBlockHeight", "err", err, "latestBlockHeight", latestBlockHeight.Int64())
```

Comment:
alternatively you can just remove .Int64() from here - a bigint should be printed correctly

Reply:
true! done